recipe/meta.yaml

```diff
@@ -252,8 +254,11 @@ outputs:
     script: install_clang_symlinks.sh  # [unix]
     script: install_clang_symlinks.bat  # [win]
     requirements:
-      - {{ pin_subpackage("clang-" ~ major_version, exact=True) }}
-      - gcc_impl_{{ target_platform }}  # [linux]
+      host:
+        - zlib  # [win]
+      run:
+        - {{ pin_subpackage("clang-" ~ major_version, exact=True) }}
+        - gcc_impl_{{ target_platform }}  # [linux]
```

Comment:
To be honest, I don't know what a build/host/run-less `requirements` defaults to and the [docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section) don't mention it either. So not sure whether
```
        - {{ pin_subpackage("clang-" ~ major_version, exact=True) }}
        - gcc_impl_{{ target_platform }}  # [linux]
```
should be host or run

Reply:
Should be both.

Reply:
It defaults to both host and run.